src/components/pages/account/AccountInfo.tsx

```diff
@@ -15,66 +14,23 @@ export type AccountInfoProps = {
 export default function AccountInfo({
   account,
 }: AccountInfoProps): JSX.Element {
-  let { data: splits } = useSplits({ guid: account.guid });
-  splits = splits || [];
+  const { data: t } = useSplitsTotal(account.guid);
 
-  const total = new Money(splits.reduce(
-    (acc, split) => acc + split.quantity,
-    0,
-  ), account.commodity.mnemonic);
-  const numMonths = (splits.length && (splits[0].transaction.date.diff(
-    splits[splits.length - 1].transaction.date,
-    ['months', 'days'],
-  ).months || 1)) || 1;
-  const average = new Money(total.toNumber() / numMonths, account.commodity.mnemonic);
-
-  let totalKeyword = 'have';
-  if (account.type === 'EXPENSE') {
-    totalKeyword = 'have spent';
-  }
-  if (account.type === 'INCOME') {
-    totalKeyword = 'have earned';
-  }
-  if (account.type === 'LIABILITY') {
-    totalKeyword = 'owe';
-  }
-
-  const currentYear = DateTime.now().year;
-  let totalThisYear = 0;
-
-  splits.every(split => {
-    if (split.transaction.date.year !== currentYear) {
-      return false;
-    }
-
-    let { quantity } = split;
-    if (account.type === 'INCOME') {
-      quantity = -quantity;
-    }
-
-    totalThisYear += quantity;
-    return true;
-  });
```

Comment:
I'm removing some useless information. We will be adding extra stuff for each account type in later PRs same as I did for Investment type